packages/client/src/components/interface/DownloadButton.tsx

```diff
@@ -139,10 +137,10 @@ function getAssignModalOptions(
       actions: [cancelAction, unassignAction]
     }
   } else if (assignment) {
-    if (
-      userRole === SystemRoleType.LocalRegistrar ||
-      userRole === SystemRoleType.NationalRegistrar
-    ) {
+    // userRole === SystemRoleType.LocalRegistrar ||
+    // userRole === SystemRoleType.NationalRegistrar
+    // @TODO: In DownloadButton, is it correct to assume that record.register can always unassign?
+    if (scopes.includes('record.register')) {
```

Comment:
@jpye-finch In DownloadButton, is it correct to assume that record.register can always unassign?

The previous condition was:
```
      userRole === SystemRoleType.LocalRegistrar ||
      userRole === SystemRoleType.NationalRegistrar
```
if it matches then your actions are `actions: [cancelAction, unassignAction]`

Reply:
Currently you can unassign by clicking the User profile image. Defined in DownloadButton i believe. 

I had it as a separate scope. record.unassign-others

Which is actioned in the Action menu.

<img width="272" alt="image" src="https://github.com/opencrvs/opencrvs-core/assets/46478402/6a53e3e5-ca89-4962-a402-114df715b64e">